website/content/api-docs/secret/ssh.mdx

```diff
@@ -817,8 +817,14 @@ $ curl \
 
 ## Sign SSH Key
 
-This endpoint signs an SSH public key based on the supplied parameters, subject
-to the restrictions contained in the role named in the endpoint.
+This endpoint signs an SSH public key based on the supplied parameters, subject 
+to the restrictions contained in the role named in the endpoint. This path 
+supports `create` and `update` policy capabilities as follows: if the user has 
+`create` capability for this endpoint, and the key does not exist, it will be 
+created with the default parameters already configured. If the user only has 
+`update` capability and the SSH key does not exist, an error will be returned. 
+To fully manage the signing of SSH keys using a single role both `create` and 
```

Comment:
```suggestion
To fully manage the signing of SSH keys using a single role, both `create` and 
```

Reply:
If it's not too much trouble, perhaps but the following in bullet form:

- if the user has `create` capability for this endpoint, and the key does not exist, it will be  created with the default parameters already configured.
- If the user only has `update` capability and the SSH key does not exist, an error will be returned.